pallets/payment/src/weights.rs

```diff
@@ -1,23 +1,25 @@
-//! Autogenerated weights for virto_payment
+
+//! Autogenerated weights for `virto_payment`
```

Comment:
Was this file really autogenerated? what's up with the karura stuff

Reply:
Yes, generated from our benchmarks, the Karura stuff was because I copied the Karura template to create a [benchmark template](fix-benchmarks-weights) but we don't really need it unless we use a CI to autogenerate weights. 

Note : the weight values don't really mean anything right now, they correspond to my local machine, we need to decide on a reference machine and generate weights on that machine. I've created a ticket here https://github.com/virto-network/virto-node/issues/179